Client/Frontend/Browser/FaviconHandler.swift

```diff
@@ -44,14 +43,13 @@ class FaviconHandler {
         let onSuccess: (Favicon, Data?) -> Void = { [weak tab] (favicon, data) -> Void in
             tab?.favicons.append(favicon)
 
-            guard !(tab?.isPrivate ?? true), let appDelegate = UIApplication.shared.delegate as? AppDelegate, let profile = appDelegate.profile else {
+            guard !(tab?.isPrivate ?? true) else {
```

Comment:
Simplify this condition while we are at it, it's unreadable atm

Reply:
While I was at it I simplified the rest of this closure lol